app/celery/letters_pdf_tasks.py

```diff
@@ -79,3 +79,45 @@ def get_letters_pdf(template, contact_block, org_id, values):
     billable_units = math.ceil(int(resp.headers.get("X-pdf-page-count", 0)) / pages_per_sheet)
 
     return resp.content, billable_units
+
+
+@notify_celery.task(name='collate-letter-pdfs-for-day')
+def collate_letter_pdfs_for_day(date):
+    letter_pdfs = s3.get_s3_bucket_objects(
+        current_app.config['LETTERS_PDF_BUCKET_NAME'],
+        subfolder=date
+    )
+    for letters in group_letters(letter_pdfs):
+        filenames = [letter['Key'] for letter in letters]
```

Comment:
Just tried it and because we are creating zip files in the same folder on FTP it's probably a good idea to make sure that only files with `.PDF` extensions should be in the array otherwise other files like zip files could be added to the zip archive. Could be argued that it will only be generated once and so it won't have the zip file in the folder but if we were to reprocess a day then the zip file might be present.